src/main/java/org/jenkinsci/plugins/nomad/NomadProvisioningStrategy.java

```diff
@@ -71,5 +83,40 @@
         LOGGER.log(Level.FINE, "Provisioning not complete, consulting remaining strategies");
         return NodeProvisioner.StrategyDecision.CONSULT_REMAINING_STRATEGIES;
     }
-}
 
+    private static void fireOnStarted(final Cloud cloud, final Label label,
```

Comment:
I'm not super familiar with the CloudProvisioningListener, but I would expect a comment like "force the onStarted event on the cloud provisioner in order to ..."

Do you think you can add a comment that would help a bit?

Reply:
Not sure my comment helps much, this is what I read - not sure who is using that event